File: 2145/python/main.py
from typing import *


class Solution:
    def prefixCount(self, words: List[str], pref: str) -> int:
        count = 0

        for word in words:

            if word[: len(pref)] == pref:
                count += 1

        return count


s = Solution()

print(s.prefixCount(["pay", "attention", "practice", "attend"], "at"))
print(s.prefixCount(words=["leetcode", "win", "loops", "success"], pref="code"))

File: 2145/typescript/main.ts
function prefixCount(words: string[], pref: string): number {
  let count = 0;

  for (const word of words) {
    if (word.substring(0, pref.length) === pref) {
      count++;
    }
  }

  return count;
}

console.log(prefixCount(["pay", "attention", "practice", "attend"], "at"));
console.log(prefixCount(["leetcode", "win", "loops", "success"], "code"));
